IFC4x3/Properties/f/FaultThickness_3pHrpKqNzD2x2$n3ipmRil/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FaultThickness_3pHrpKqNzD2x2$n3ipmRil" Name="FaultThickness" UniqueId="f3475cd4-d17f-4d0b-b0bf-c43b33c1bb2f" PropertyType="p_boundedvalue" PrimaryDataType="IfcNonNegativeLengthMeasure" />
```

Comment:
Bounded value?

FaultThickness can be 0?

Reply:
single or bounded value, no standard

Reply:
> Bounded value?
> 
> FaultThickness can be 0?

In excel: IfcPositiveLengthMeasure is striked through (over) and IfcNonNegativeLengthMeasure is added in red :-)

Reply:
> single or bounded value, no standard

This does not help much. Can be either single or bounded.

Reply:
Updated description.